Examples/ExamplesApp/Examples.swift

```diff
@@ -83,27 +93,26 @@ extension Category {
     }
 #endif
     
-    static var geoview: Self {
+    /// Components requiring specific data.
+    static var data: Self {
         .init(
-            name: "GeoView",
+            name: "Data",
```

Comment:
Would "Advanced", "Specialty", or "Specialized" be better for this category?

Reply:
Data is OK for now.